tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -132,13 +133,19 @@ def test_check_mds_cache_memory_limit(self):
         Testcase to check mds cache memory limit post ocs upgrade
 
         """
-        mds_cache_memory_limit = get_mds_cache_memory_limit()
-        expected_mds_value = 4294967296
-        expected_mds_value_in_GB = int(expected_mds_value / 1073741274)
-        assert mds_cache_memory_limit == expected_mds_value, (
-            f"mds_cache_memory_limit is not set with a value of {expected_mds_value_in_GB}GB. "
-            f"MDS cache memory limit is set : {mds_cache_memory_limit}B "
-        )
-        log.info(
-            f"mds_cache_memory_limit is set with a value of {expected_mds_value_in_GB}GB"
-        )
+        for sampler in TimeoutSampler(
+            timeout=900, sleep=60, func=get_mds_cache_memory_limit
+        ):
+            if sampler:
+                mds_cache_memory_limit = get_mds_cache_memory_limit()
+                expected_mds_value = 4294967296
+                expected_mds_value_in_GB = int(expected_mds_value / 1073741274)
+                assert mds_cache_memory_limit == expected_mds_value, (
```

Comment:
is it not better to set the values as constants ?

Reply:
agreed, will fix that